app/src/main/jni/desmume/src/MMU.cpp

```diff
@@ -49,22 +49,22 @@
 #define FLOATING_SQRT
 
 #ifdef HAVE_NEON
+#if defined(__arm__)
```

Comment:
I removed the "HAVE_NEON" from ARMv8 long ago, but that condition was still being fulfilled. That's why that line of code is there.